colin/checks/labels/distribution-scope.py

```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class DescriptionScopeLabelCheck(LabelCheck):
```

Comment:
This is set automatically, based on the our document. I don't know if it make sense to have it in Fedora.